go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/nginxinc/telemetry-exporter
 
-go 1.22.4
```

Comment:
I agree with @danielnginx this should not be downgraded, the purpose of this bump is to consume Go stdlib vuln fixes.

Please reach out to Nginx security team with any questions.

Reply:
Resolving after internal discussion on this.